.github/workflows/ci.yml

```
@@ -0,0 +1,19 @@
+name: Continuous Integration
+on: [push]
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        version: [ '7.1', '7.2', '7.3', '7.4', '8.0' ]
```

Maybe we can add `8.1` if the code is compatible ?

```suggestion
        version: [ '7.1', '7.2', '7.3', '7.4', '8.0', '8.1' ]
```